index.html

```diff
@@ -186,28 +196,33 @@ <h2 class="title">About Myself</h2>
                     </li>
 
                 </ul>
-                <button class="button" type="button"><b>Get my resume</b></button>
-                <ul class="features">
-                    <li class="second_menu">
-                        <p class="languages">Languages</p>
-                        <img class="arrow_icon arrow_icon_active" src="images/icon_arrow.png" alt="arrow">
-
-                    </li>
-                    <ul class="languages_content">
-                        <li><img src="images/js_icon.png" alt="javascript"><b class="language">JavaScript</b></li>
-                        <li><img src="images/html_icon.png" alt="html"><b class="language">HTML</b></li>
-                        <li><img src="images/css_icon.PNG" alt="css"><b class="language">CSS</b></li>
+                <button class="button bottom" type="button"><b>Get my resume</b></button>
+                </div>
+                <div class="about_me_2">
+                    <ul class="features">
+                        <li class="second_menu">
+                            <p class="languages">Languages</p>
+                            <img class="arrow_icon arrow_icon_active" src="images/icon_arrow.png" alt="arrow">
+    
+                        </li>
+                        <ul class="languages_content">
+                            <li><img src="images/js_icon.png" alt="javascript"><b class="language">JavaScript</b></li>
+                            <li><img src="images/html_icon.png" alt="html"><b class="language">HTML</b></li>
+                            <li><img src="images/css_icon.PNG" alt="css"><b class="language">CSS</b></li>
```

Comment:
Please consider reviewing the path for this icon as it's not showing on the webpage.

## Demo:-

![image](https://user-images.githubusercontent.com/94223367/228927247-0f51f19b-0821-4f21-9720-e003c21e15b9.png)